docs/sources/datasources/grafana-pyroscope.md

```diff
@@ -30,67 +30,86 @@ To configure basic settings for the data source, complete the following steps:
 
-Query editor gives you access to a profile type selector, a label selector, and collapsible options.
+![Query editor](/media/docs/pyroscope/query-editor/query-editor.png 'Query editor')
 
-![Profile selector](/static/img/docs/phlare/select-profile.png 'Profile selector')
+To access the query editor:
 
-Select a profile type from the drop-down menu. While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid. Grafana does not show any data if the profile type or app isn’t selected when a query is run.
+1.  Sign into Grafana or Grafana Cloud.
+1.  Select your Pyroscope data source.
+1.  From the menu, choose **Explore**.
 
-![Labels selector](/static/img/docs/phlare/labels-selector.png 'Labels selector')
+1.  Select a profile type from the drop-down menu.
 
-Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels. Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
+    {{< figure src="/media/docs/pyroscope/query-editor/select-profile.png" class="docs-image--no-shadow" max-width="450px" caption="Profile selector" >}}
 
-![Options section](/static/img/docs/phlare/options-section.png 'Options section')
+1.  Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels.
+    Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
 
-Options section contains a switch for Query Type and Group by.
+        While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid.
 
-Select a query type to return the profile data which can be shown in the [Flame Graph][flame-graph], metric data visualized in a graph, or both. You can only select both options in a dashboard, because panels allow only one visualization.
+        Grafana doesn't show any data if the profile type or app isn’t selected when a query runs.
 
-Group by allows you to group metric data by a specified label. Without any Group by label, metric data is aggregated over all the labels into single time series. You can use multiple labels to group by. Group by has only an effect on the metric data and does not change the profile data results.
+    ![Labels selector](/media/docs/pyroscope/query-editor/labels-selector.png 'Labels selector')
+
+1.  Expand the **Options** section to view **Query Type** and **Group by**.
+    ![Options section](/media/docs/pyroscope/query-editor/options-section.png 'Options section')
+
+1.  Select a query type to return the profile data which can be shown in the [Flame Graph][flame-graph], metric data visualized in a graph, or both. You can only select both options in a dashboard, because panels allow only one visualization.
```

Comment:
is there supposed to be a link here?

Reply:
This is a less common version of Markdown link known as [_reference-style_](https://spec.commonmark.org/0.30/#link-reference-definitions).

The destination is defined in https://github.com/grafana/grafana/pull/81215/files/4eb82167a07ec87480c6330fd32801203de56679#diff-74748b5f590531c97ca006dcf25af41006921b78039b7fe41754e62f3cd66a3eR129-R130.